EdFi.Buzz.Installer/eng/Windows/Application/appinstallValidations.psm1

```diff
@@ -0,0 +1,32 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+#Requires -version 5
+#Requires -RunAsAdministrator
+
+function Authentication-Configuration-Valid {
```

Comment:
PowerShell is complaining about the verb name, and the extra hyphen. Usually PowerShell functions have a Verb-Phrase format - Install-BuzzApp. And the verb should be approved.

I would recommend just globally changing Authentication-Configuration-Valid to Test-AuthConfiguration everywhere. That should get around it...

You can check the PS verbs with this command....
`Get-Verb | ? { $_.Verb -like '*test*'}`

Reply:
Also, appinstall.ps1 has `Check-BuzzServices`. Check is not an approved verb, 

We should also change that to `Test-BuzzServices`.

Reply:
Right... I saw the warning and I thought: I'll deal with that later. And then I forgot.  Thanks!

Reply:
Fixed.